pulsar-broker/src/test/java/org/apache/pulsar/client/impl/MessageChunkingSharedTest.java

```diff
@@ -211,6 +213,49 @@ private static String createChunkedMessage(int numChunks) {
         return Schema.STRING.decode(payload);
     }
 
+    @Test
```

Comment:
Also, 
if feasible, please add other edge case tests.
- broker restarts
- msg persists on bk fails

Reply:
As far as the new implementation is concerned, we don't need to do these tests.